dashboard/config/scripts/pre-express-2020.script

```diff
@@ -137,23 +137,7 @@ level 'courseB_collector_loops11_2020'
 bonus 'courseB_collector_loops_challenge1_2020'
 bonus 'courseB_collector_loops_challenge2a_2020'
 
-lesson 'Drawing Gardens with Loops'
-level 'courseB_video_ArtistIntro_2020'
-level 'courseB_artist_loops1_2020'
-level 'courseB_video_artistLoops_2020'
-level 'courseB_artist_loops2_2020'
-level 'courseB_artist_loops3_2020'
-level 'courseB_artist_loops4_2020'
-level 'courseB_artist_loops5_2020'
-level 'courseB_artist_loops6_2020'
-level 'courseB_artist_loops7_2020', challenge: true
-level 'courseB_artist_loops8_2020'
-level 'courseB_artist_loops9_2020'
-level 'courseB_artist_loops10_2020'
-bonus 'courseB_artist_loops_challenge1_2020'
-bonus 'courseB_artist_loops_challenge2_2020'
-
-lesson 'Ocean Scene with Loops'
+lesson 'Ocean Scene with Loops', display_name: 'Ocean Scene with Loops'
```

Comment:
This change is correct. These lessons were out of order. I have double checked it on Curriculum Builder multiple times.